test/unit/os/shell_sxe_sxq_spec.lua

```diff
@@ -0,0 +1,43 @@
+-- not all operating systems support the shell_build_argv tests, as of yet.
+local allowed_os = {
+  Linux = true,
+  OSX = true,
+  BSD = true,
+  POSIX = true,
+  Windows = true
```

Comment:
Not sure, but are there any OS that are *not* supported here? Otherwise this block can just be dropped.